XamlStyler.Mac/Gui/XamlStylerOptionsPanel.cs

```diff
@@ -1,19 +1,26 @@
-﻿using MonoDevelop.Ide.Gui.Dialogs;
+﻿using MonoDevelop.Components;
+using MonoDevelop.Ide.Gui.Dialogs;
+using Xavalon.XamlStyler.Mac.ViewModels;
 
 namespace Xavalon.XamlStyler.Mac.Gui
 {
 	public class XamlStylerOptionsPanel : OptionsPanel
 	{
-		private OptionsViewModel _optionsViewModel = new OptionsViewModel();
+        public XamlStylerOptionsPanel()
+        {
+            ViewModel = new XamlStylerOptionsViewModel();
+        }
 
-		public override MonoDevelop.Components.Control CreatePanelWidget()
+        private XamlStylerOptionsViewModel ViewModel { get; }
+
+		public override Control CreatePanelWidget()
```

Comment:
This code is not aligned with the rest of the file

Reply:
should be aligned now